test/CMakeLists.txt

```diff
@@ -36,7 +35,7 @@ endif()
 ## Lists tests
 set(tests aabb2 access_restriction actor admin attributes_controller configuration datetime directededge
   distanceapproximator double_bucket_queue edgecollapser edgestatus ellipse encode
-  enhancedtrippath factory graphid graphtile graphtileheader gridded_data grid_range_query grid_traversal instructions
+  enhancedtrippath factory geometry graphid graphtile graphtileheader gridded_data grid_range_query grid_traversal instructions
```

Comment:
randomly found out that this test was missing in the list. should increase coverage:)

Reply:
But also goals the arm build :(